app/javascript/dashboard/components/widgets/WootWriter/FullEditor.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <div>
     <div class="editor-root editor--article">
+      <input
+        ref="imageUploadInput"
+        type="file"
+        accept="image/png, image/jpeg, image/jpg, image/gif"
```

Comment:
```suggestion
        accept="image/*"
```

We could use this to accept all kinds of images, unless we're limiting to these for a specific reason. Or we could add `image/webp`

Reply:
I see in the previous conversation, SVG support is broken. Let's just add `image/webp` then. Updating suggestion

Reply:
@scmmishra I added it in this PR. The reason is if we add an SVG in the article editor, It looks broken. You can see it in this video.
https://user-images.githubusercontent.com/64252451/217842408-efb3b672-f10e-404a-908a-ba24380eae97.mov

Reply:
Noted, let's just allow webp too